packages/lib/src/vis/line/ErrorBars.tsx

```diff
@@ -17,31 +18,33 @@ interface Props {
 }
 
 function ErrorBars(props: Props) {
-  const { abscissas, ordinates, errors, color, visible, ignoreValue } = props;
-  const { abscissaScale, ordinateScale } = useVisCanvasContext();
-
-  const geometryParams = {
+  const {
     abscissas,
     ordinates,
     errors,
-    abscissaScale,
-    ordinateScale,
+    color,
+    visible = false,
```

Comment:
Very subtle fix here: without this default value, I was seeing a subtle empty error bar in the middle of the canvas:

<img width="55" height="57" alt="image" src="https://github.com/user-attachments/assets/0e62cfe0-c899-4c96-a450-65fa5789ed3e" />